### DISCUSSION TITLE
能不能缓存上次的播放进度到本地

### DISCUSSION THREAD

**Discussion by Ychan98** [the1812/Bilibili-Evolved#3690] [功能建议 / Ideas]
自带的记忆进度老是莫名其妙的丢，昨天看的视频今天点进去也不能跳转上次进度，所以能不能比如本地缓存100个视频播放进度，然后稍后再看列表里的视频进度永远保存这样的功能

**Comment by Ethkuil** [comment]
这个似乎更偏向于是B站官方的工作？个人看法

**Reply by Geeyun-JY3** [reply]
楼主的意思就是破站的这个记忆做得不行
